### PR TITLE
add curation policy

### PR DESCRIPTION
**Pull request by bwhm**

### CODE REVIEW

File: roles/content-curator-lead/README.md [hunk 1]
```diff
@@ -207,7 +208,7 @@ Other than the hiring aspect of the role as `Content Curator Lead`, the lead sho
 
 If necessary, upon discussing with the council, the `Content Curator Lead` can also decide to fire curators who are not performing their jobs adequately.
 
-Most of the time however, the responsibilities of the `Content Curator Lead` will be very similar to those of a standard `Content Curator`. You can read about these responsibilities in [this section](../roles/content-curators#content) of the guide for `Content Curators`.
+Most of the time however, the responsibilities of the `Content Curator Lead` will be very similar to those of a standard `Content Curator`. You can read about these responsibilities in [this section](/roles/content-curators#content) of the guide for `Content Curators`, but will in the end be held responsible for the reporting and work performed by their team.
```
Comment (blrhc) [inline roles/content-curator-lead/README.md:211]:
```suggestion
Most of the time however, the responsibilities of the `Content Curator Lead` will be very similar to those of a standard `Content Curator`. You can read about these responsibilities in [this section](/roles/content-curators#content) of the guide for `Content Curators`, but will in the end be held responsible for the reporting and work performed by their team.
```
```suggestion
Most of the time however, the responsibilities of the `Content Curator Lead` will be very similar to those of a standard `Content Curator`. You can read about these responsibilities in [this section](/roles/content-curators#content) of the guide for `Content Curators`, but note that the Lead also has the ultimate responsibility for the reporting and work performed by their team.
```

File: roles/content-curator-lead/README.md [hunk 2]
```diff
@@ -246,6 +247,22 @@ To check the details of the current Content Working Group Mint:
...
+
+This is done using [Extrinsics](https://testnet.joystream.org/#/extrinsics), with `contentDirectory.updateEntityCreationVoucher`.
+Example:
+If member `1` is to be given the right to make 200 vidoes:
+`class_id`: 10
+`controller`: Member
+`Member`: 1
+`maximum_entities_count`: 200
+
+Only the Leads "role" key can perform this transaction, and any other origin will be rejected.
```
Comment (blrhc) [inline roles/content-curator-lead/README.md:263]:
```suggestion
Only the Lead's "role" key can perform this transaction, and any other origin will be rejected.
```

File: roles/content-curators/README.md [hunk 3]
```diff
@@ -96,27 +101,189 @@ If your application was successful, you will now be able to curate content on th
 
 The main job for curators is to regularly check the content directory and channels, and checking that rules and guidelines are being followed.
 
-## Rules
-The basic rules are stated below.
+## Curation Policy
+Due to the large influx of users, and incentives for video curation, we need a formal workflow for the `Content Curators`, and a curation policy.
+
+Note that parts of this policy only applies to this testnet.
```
Comment (blrhc) [inline roles/content-curators/README.md:107]:
```suggestion
Note that parts of this policy only applies to this testnet (Babylon).
```

File: roles/content-curators/README.md [hunk 4]
```diff
@@ -96,27 +101,189 @@ If your application was successful, you will now be able to curate content on th
...
+
+`censor`
+Means updating the status (`isCensored`) for a video from `true` to `false`. Unlike `hidden`, this can not be changed back by anyone but another curator.
+
+#### Videos
+**When to (only) issue a `warning`:**
+- Missing artwork
+  - If the thumbnail is missing, or just really poor, the video can be `hidden`, or issued a `warning` (Council decides u.n.o.).
+- Suspicious license
+  - If the Curator suspects, but is not able to ensure, a video is incorrectly licensed, a `warning` can be issued, leaving the channel owner some defined deadline to respond.
```
Comment (blrhc) [inline roles/content-curators/README.md:245]:
```suggestion
  - If the Curator suspects, but is not able to verify that a video is incorrectly licensed, a `warning` can be issued, leaving the channel owner some defined deadline to respond and provide more information or evidence.
```

File: roles/content-curators/README.md [hunk 5]
```diff
@@ -96,27 +101,189 @@ If your application was successful, you will now be able to curate content on th
...
+- The content must be as "advertised"
+  - If the title, description, category and thumbnail implies a baking video, the video should not be a documentary about Bitcoin
+- Missing artwork
+  - If thumbnail is missing, or just really poor, the the video can be hidden, or given a warning (Council decides u.n.o.)
+- Poor quality
+  - If the video quality is "unreasonably" low
+
+**When to `censor` a video:**
+- License requires attribution
+  - If the selected license requires attribution, but non is given the, or attribution is incorrect
```
Comment (blrhc) [inline roles/content-curators/README.md:257]:
```suggestion
  - If the selected license requires attribution, but none is given the, or attribution is incorrect
```

File: roles/content-curators/README.md [hunk 6]
```diff
@@ -96,27 +101,189 @@ If your application was successful, you will now be able to curate content on th
...
+- Suspected copyright violation
```
Comment (blrhc) [inline roles/content-curators/README.md:258]: Perhaps add above new bullet point `Any other breach of License terms`

File: roles/content-curators/README.md [hunk 7]
```diff
@@ -96,27 +101,189 @@ If your application was successful, you will now be able to curate content on th
...
+  - If it's not clear that the video is in violation, but this is strongly suspected by the Curator
+- Status change by owner without resolving issue
+  - If a video is `hidden` by a curator, and the channel owner makes it public again without correcting the issue described by the curators
+- Terms of Service violation
+  - If the video is in violation of other parts of the [ToS](https://testnet.joystream.org/#/pages/tos)
+
+#### Channels
+**When to (only) issue a `warning`:**
+- Missing artwork
+  - If the avatar or cover is missing, or just really poor, the channel can be `hidden`, or issued a `warning` (Council decides u.n.o.).
```
Comment (blrhc) [inline roles/content-curators/README.md:268]:
```suggestion
  - If the avatar or cover is missing, or just really poor, the channel can be `hidden`, or issued a `warning` (Council decides).
```

Comment (blrhc) [reply roles/content-curators/README.md:268]: Council decides?
Shouldn't it be curators?

**Review by blrhc — CHANGES_REQUESTED** [review]
Couple of suggestions.

File: roles/content-curators/README.md (hunk 7 above)
Comment (bwhm) [reply roles/content-curators/README.md:268]: I see your point, they have the hammer! However, the Council should be able to set overarching policies such as this.

**Review by blrhc — APPROVED** [review]
Looks good!

**Pull request merged**